packages/web/src/components/spa/HackAUBG/FAQSection/faq_section.module.css

```diff
@@ -0,0 +1,64 @@
+.faq {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    left: 50%;
+    transform: translate(-50%, 0);
+    z-index: 100;
+    margin-bottom: 10%;
+    align-items: center;
+}
+
+.faq-h1 {
+    display: flex;
+    color: #ffffff;
```

Comment:
add align-itmes center because at some resolutions the title is not centred properly